modules/ocf_mirrors/templates/record-mirrors-stats.py.erb

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+
+import os
+import pymysql
+from datetime import date
```

Comment:
group imports into stdlib / 3rd party / application per PEP8: https://www.python.org/dev/peps/pep-0008/#imports

pre-commit should enforce this for you?